sys-fs/genfstab/genfstab-28-r1.ebuild

```diff
@@ -22,3 +24,9 @@ BDEPEND="
 src_test() {
 	emake check
 }
+
```

Comment:
you could do a src_compile() with MANS="doc/genfstab.8" BINPROGS="genfstab" to only build genfstab

Reply:
Thanks, I didn't notice this!

Reply:
```
src_compile() {
        emake MANS="doc/genfstab.8" BINPROGS="genfstab"
}

src_install() {
        dobin genfstab
        doman doc/genfstab.8
        newbashcomp completion/genfstab.bash genfstab
}
```

I will change the default compile block. But still use `dobin`/`doman`/`newbashcomp` in install block, because `emake MANS="doc/genfstab.8" BINPROGS="genfstab" install ` will install all completions file, not only genfstab.

Do you think this is okay?

Reply:
ah yea that should be good